tools/spark-reader-v1/src/test/java/com/vesoft/nebula/NebulaReaderExample.java

```diff
@@ -0,0 +1,62 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+package com.vesoft.nebula;
+
+import com.facebook.thrift.protocol.TCompactProtocol;
+import com.vesoft.nebula.common.Type;
+import org.apache.spark.SparkConf;
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.SparkSession;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class NebulaReaderExample {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(NebulaReaderExample.class);
+
+    public static void main(String[] args) {
+        SparkConf sparkConf = new SparkConf();
+        sparkConf.set("spark.serializer", "org.apache.spark.serializer.KryoSerializer");
+        sparkConf.registerKryoClasses(new Class[]{TCompactProtocol.class});
+        SparkSession sparkSession = SparkSession
+                .builder()
+                .config(sparkConf)
+                .master("local[4]")
+                .getOrCreate();
+
+        Dataset<Row> vertexDataset = sparkSession
+                .read()
+                .format("com.vesoft.nebula.reader.NebulaDataSource")
+                .option("importType", Type.VERTEX.getType())
+                .option("ip", "127.0.0.1")
```

Comment:
why don't merge ip and port into one for example host0:port0,host1:port1

Reply:
your suggestion is better, i will changge this part of code